md/run-a-process-from-bonita-bpm-studio-for-testing.md

```diff
@@ -8,7 +8,10 @@ process development.
 Before you can run a process, you must [configure the process](configuring-a-process.md)
 and [configure the organization](organization-management-in-bonita-bpm-studio.md).
 
-To run a process, open the process diagram and select the pool, then click **_Run_** in the Cool bar. The process in the selected pool runs. Only one pool can be run at a time.
+
+To run a process, you can :
+* Right click on the process diagram on the project explorer, and select the process to run from the **run** menu.
```

Comment:
same remark

Reply:
Run a pool sounds weird in my opinion.

Reply:
https://documentation.bonitasoft.com/bonita/7.7/run-a-process-from-bonita-bpm-studio-for-testing
We already talk about running 'process from the Studio' in the documentation, so it's fine I think.